### DISCUSSION TITLE
What could be causing "Session Likely Expired" error?

### DISCUSSION THREAD

**Discussion by tt0mmy** [Web3Auth/Web3Auth#760] [Help]
I'm seeing this error that says "Session likely expired" and not sure why.  This seems to appear as soon as `web3auth.init` is completed.  I'm still able to call `await web3auth?.connectTo()` and login successfully even when the error does appear.  Any concerns here?  This is happening on `testnet` haven't tested on `cyan` or `mainnet` yet.

```
    "@web3auth/openlogin-adapter": "^2.1.3",
    "@web3auth/web3auth": "^2.1.3",
```

![image](https://user-images.githubusercontent.com/7282254/197637440-d1bad5f4-eb18-4fe1-96a3-33ca2150ad67.png)

**Comment by himanshuchawla009** [comment]
Hello, this is just a warning, you can ignore it. SDK first creates a new session using a random session id and then checks if the same sessionId already exists, if it doesn't exist it shows a warning which can be ignored. It should not block user's login.
